README.md

```diff
@@ -704,6 +704,20 @@ Use the same for main `tarantool` package and `queue` and `uuid` subpackages.
 `uuid` tests require
 [Tarantool 2.4.1 or newer](https://github.com/tarantool/tarantool/commit/d68fc29246714eee505bc9bbcd84a02de17972c5).
 
+To run benchmark tests, call
+```bash
+make bench DURATION=100x
+```
+
+Use the variable `DURATION` to set the duration of perf tests. That variable is mapped on
+testing [flag](https://pkg.go.dev/cmd/go#hdr-Testing_flags) `-benchtime` for gotest. It is set to `2s` by default.
+
+To run ab perfomance tests, call
```

Comment:
Probably it is worth to explain what `ab` means here.

Reply:
Fixed.